src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -195,9 +196,12 @@ package com.google.example.examples.library.v1;
 
 // [START sample]
 import com.google.example.library.v1.FindRelatedBooksRequest;
+import com.google.example.library.v1.FindRelatedBooksResponse;
```

Comment:
Unrelated to this PR, but we should follow up separately: this particular value set defines a default value of "Odyssey", which I don't see mentioned anywhere in this sample. Other samples do have it.

Could you file an issue and we pursue this separately from this PR?

Reply:
Sure.